use_cases/gretel.json

```diff
@@ -35,6 +35,25 @@
         "link": "https://colab.research.google.com/github/gretelai/gretel-blueprints/blob/main/docs/notebooks/transform_relational_database.ipynb"
       }
     },
+    {
+      "gtmId": "use-case-tabular-dp",
+      "title": "Create provably private versions of sensitive data",
+      "description": "Use Gretel Tabular DP, our new differentially private graph-based generative model, to generate synthetic data with strong differential privacy guarantees.",
+      "cardType": "Console",
+      "imageName": "tabular-dp.png",
+      "modelType": "synthetics",
+      "modelCategory": "synthetics",
+      "tag": "New",
+      "defaultConfig": "config_templates/gretel/synthetics/tabular-differential-privacy.yml",
+      "sampleDataset": {
+        "fileName": "bank_marketing_small.csv",
+        "description": "This dataset contains direct marketing campaign details (phone calls) from a Portuguese financial institution. It has sensitive information such as demographics and financials, which need to be made private before the data can be shared.  ",
```

Comment:
suggest something like `...financials, which can benefit from privacy preserving techniques before sharing`. Made private is kinda vague?